extensions-c9s.yaml

```diff
@@ -74,3 +74,12 @@ extensions:
       - sig-virtualization
     packages:
       - kata-containers
+  # https://issues.redhat.com/browse/COS-2402
+  kernel-64k:
+    architectures:
+      - aarch64
+    packages:
+      - kernel-64k-core
+      - kernel-64k-modules
+      - kernel-64k-modules-core
+      - kernel-64k-modules-extra
```

Comment:
Nit: You need an empty line at the end of a file. Make sure to configure your editor to do it for you. In some editors, this empty line is not shown, in some it is.